packages/proteus/src/start/startBenchmark.ts

```diff
@@ -26,7 +26,7 @@ import {PreKeyBundle} from '../keys/PreKeyBundle';
 import {IdentityKeyPair} from '../keys/IdentityKeyPair';
 import {PreKey} from '../keys/PreKey';
 import {initProteus} from '../initProteus';
-import * as program from 'commander';
+import {program} from 'commander';
```

Comment:
I believe I have fixed this issue in a previous commit, please rebase or merge :)
https://github.com/wireapp/wire-web-core/commit/4188a6ee3ac88d729214d1ad80f68474be027b6c

Reply:
nvmd